dashboard/app/controllers/level_sources_controller.rb

```diff
@@ -100,6 +100,7 @@ def set_level_source
     else
       @level_source = LevelSource.where(hidden: false).find(params[:id])
     end
+    @level_source.try(:replace_old_when_run_blocks)
```

Comment:
Arguably we shouldn't need to worry about the shared level scenario, since those should have already been cleaned up by our initial migration, but this seems simple enough just in case.